src/Template/Layout/default.ctp

```diff
@@ -15,38 +9,26 @@ echo $this->fetch(
             <?= $this->fetch('title', read('App.title', env('HTTP_HOST'))) ?>
         </title>
 
+        <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta name="author" content="<?= read('App.author') ?>">
+        <?= $this->Html->meta('icon') ?>
         <?= $this->fetch('meta') ?>
-
-
         <?= $this->AssetCompress->css('platform') ?>
-        <?= $this->fetch('css'); ?>
-        <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
+        <?= $this->fetch('css') ?>
+        <?= $this->fetch('headjs') ?>
+
+        <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
```

Comment:
Is that really a good idea ... Looking at the browser shares even supporting Android 2 default Browser could have a bigger impact?